includes/news/text/base.php

```diff
@@ -2,52 +2,50 @@
 UCF TODAY
 <?php echo date( 'l, F j, Y' ) ?>
 
-<?php $weather = get_weather( 'weather-today' ); if ( !empty( $weather ) ) { ?>
+<?php 
+$weather = get_weather( 'weather-today' ); 
+if ( !empty( $weather ) ) : ?>
 High: <?php echo $weather['today']['tempN']; ?> Low: <?php echo $weather['tonight']['tempN'] ?>
-<?php } ?>
+<?php endif; ?>
 
-<?php $top_story_details = get_top_story_details(); ?>
 
 
-TODAY'S TOP STORIES
 
-<?php echo strip_tags( $top_story_details['story_title'] ); ?>
-
-<?php echo strip_tags( $top_story_details['story_description'] ); ?>
-
-<?php echo $top_story_details['read_more_uri']; ?>
-
-
-<?php foreach( get_featured_stories_details( 4 ) as $detail ) { ?>
-<?php echo strip_tags( $detail['title'] ); ?>
-
-<?php echo strip_tags( $detail['description'] ); ?>
-
-<?php echo $detail['permalink']; ?>
-
-
-<?php } ?>
+<?php
+$gmucf_content = get_gmucf_email_options_feed_values();
+$send_date     = $gmucf_content->gmucf_email_send_date;
 
+if ( $send_date === date( 'm/d/Y' ) ) {
+    echo get_template_part( 'includes/news/text/email-text' );
+} else {
+    echo get_template_part( 'includes/news/text/backup-email-text' );
+}
+?>
 
 UCF IN THE NEWS
+========================
+<?php foreach( get_in_the_news_stories() as $story ) : ?>
+- <?php echo strip_tags( $story->link_text ); ?> (<?php echo trim( strip_tags( $story->source ) ); ?>)
```

Comment:
If the story source is not set, you end up with an empty set of parentheses here (see Dev, "The Newest External Story ()").  Would recommend making sure the trimmed/stripped `$story->source` is not empty before echoing here